shex/sio-model/diseaseHistoryAndDiagnosisShape.shex

```diff
@@ -19,19 +18,32 @@ PREFIX loinc: <http://purl.bioontology.org/ontology/LNC/>
 }
 
 :processShape IRI {
-  a [sio:SIO_000006 sio:SIO_001001];
-  prov:wasInformedBy @:processShape ?;
-  prov:startedAtTime xsd:dateTime;
+  (a [sio:SIO_000006];
+  sio:SIO_000230 @:outputReferToDiseaseShape;
   sio:SIO_000229 @:outputShape
+  |
+  a [sio:SIO_001001];
+  sio:SIO_000229 @:outputReferToDiseaseShape
+  );
+  sio:SIO_000680 xsd:dateTime
 }
 
 :outputShape IRI {
   a [sio:SIO_000340];
-  (a [loinc:LP91394-4]|
+  a [loinc:LP91394-4]
```

Comment:
This should be list of terms. See the previous version https://github.com/ejp-rd-vp/CDE-semantic-model/blob/develop/shex/diseaseHistoryAndDiagnosisShape.shex#L36

Reply:
Thanks for identifying that. Finish updating the type.